pvlib/irradiance.py

```diff
@@ -1620,8 +1621,8 @@ def disc(ghi, solar_zenith, datetime_or_doy, pressure=101325,
         Day of year or array of days of year e.g.
         pd.DatetimeIndex.dayofyear, or pd.DatetimeIndex.
 
-    pressure : None or numeric, default 101325
-        Site pressure in Pascal. If None, relative airmass is used
+    pressure : numeric, default 101325
+        Site pressure in Pascal. If not specified, relative airmass is used.
```

Comment:
I think the text below would clear up my confusion. I don't know if the alternative "If `None`" requires changing the parameter descriptor to `numeric or None`,  

```suggestion
        Site pressure in Pascal. If numeric, absolute airmass is used.
        If ``None``, relative airmass is used.
```

Reply:
English is not my mother tongue, but I really like how it currently is. Anyways, we could change "If None" to "Otherwise, ...". I don't dislike your suggestion.

Or to "If not provided". I think the latter makes more sense. It's up to you.

Reply:
My problem is with `numeric, default 101325`. So it has a value (a number). If I don't provide a different value, 101325 is used. Then what triggers the "not specified" condition?

Maybe it would be better left as original, but perhaps `numeric or None, default 101325`.

Reply:
Ah, I see... from the coded behaviour it needs to be `None` to use relative AM. Also, see [current docs](https://github.com/pvlib/pvlib-python/blob/57d61af5aba87e7089a78cfae0b26d153a130fc7/pvlib/irradiance.py#L1604-L1610). `pressure=None` is mentioned. All I see are reasons to stick to the original. Thanks for spotting that.

Reply:
How do you feel about this text?

```
pressure : numeric or None, default 101325
        Site pressure in Pascal. Uses absolute (pressure-corrected) airmass
        by default.
        Set to ``None`` to use relative airmass.
```

I think it's safe to assume that if someone inputs the pressure it means they will know that it is using it to compute absolute airmass. Or we may clear that up with:

```
pressure : numeric or None, default 101325
        Site pressure in Pascal. Values to compute absolute
        (pressure-corrected) airmass. Uses sea-level pressure by default.
        Set to ``None`` to use relative airmass.
```

A bit more descriptive than it currently is IMO.

Reply:
I like the first, that is clear to me. Thanks for your patience.